modules/ossm-rn-known-issues.adoc

```diff
@@ -198,6 +201,14 @@ New issues for Kiali should be created in the link:https://issues.redhat.com/pro
 
 These are the known issues in Kiali:
 
+* https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click on the **Node graph** menu option of any node menu within the traffic graph, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. There is currently no workaround for this issue.
+
+* https://issues.redhat.com/browse/OSSM-6298[OSSM-6298] When you click on an item reference within the {SMPlugin}, such as a workload link related to a specific service, the console sometimes performs multiple redirections before landing on the desired page. If you click the "Back" button in the web browser, the console navigates to a different page of the console instead of the previous page.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-6298[OSSM-6298] When you click an item reference within the {SMPlugin}, such as a workload link related to a specific service, the console sometimes performs multiple redirections before opening the desired page. If you click *Back* in a web browser, the a different page of the console opens instead of the previous page.
```
* Take a look at https://www.ibm.com/docs/en/ibm-style?topic=elements-ui to review the guidelines for referring to UI elements.